collectors/cgroups.plugin/sys_fs_cgroup.c

```diff
@@ -3697,6 +3724,7 @@ void update_cgroup_charts(int update_every) {
                 );
 
                 rrdset_update_rrdlabels(cg->st_cpu, cg->chart_labels);
+                rrdlabels_add(cg->st_cpu->rrdlabels, "_cgroup_name", cg->chart_id, RRDLABEL_SRC_AUTO);
```

Comment:
Why the label name starts with `_`?

Reply:
It was because we were using this for internal label. I removed.